chrome.ts

```diff
@@ -238,7 +238,7 @@ class ChromeImpl implements Chrome {
         return;
       }
 
-      const folder = entry.folder;
+      const folder;
```

Comment:
Sorry, there was a mistake in the code I posted in the previous comment https://github.com/uki00a/carol/pull/49#discussion_r542377751.
```suggestion
      const folder = entry.folder;
```